Makefile.in

```diff
@@ -61,6 +61,8 @@ RKT_STAGE1_DEFAULT_IMAGES_DIRECTORY_LDFLAGS := @RKT_STAGE1_DEFAULT_IMAGES_DIRECT
 RKT_VERSION_LDFLAGS := @RKT_VERSION_LDFLAGS@
 RKT_FEATURES_LDFLAGS := @RKT_FEATURES_LDFLAGS@
 
+KVM_HV_TAG := "@KVM_HV_TAG@"
```

Comment:
Tested right now. Looks like it's not necessary since we include it in RKT_TAGS.